lib/webpacker/helper.rb

```diff
@@ -137,6 +141,9 @@ def preload_pack_asset(name, **options)
   #   <%= stylesheet_pack_tag 'calendar' %>
   #   <%= stylesheet_pack_tag 'map' %>
   def stylesheet_pack_tag(*names, **options)
+    css_inline = Webpacker.dev_server.hmr? && Webpacker.dev_server.configured_and_running?
+    return "" if css_inline
+
```

Comment:
@dhh and @rossta, does the above make it clear when to not have the style tag if we want CSS inlined?

Reply:
LGTM

Reply:
I would pull this up into Webpacker, so it's `Webpacker.inlining_css?`.

Reply:
Done. How's this look?